addons/io/org.openhab.io.azureiothub/README.md

```diff
@@ -1,22 +1,23 @@
-# azure IOT Hub Connector
+# Azure IoT Hub Connector
 
-The azure IOT Hub connector replicates your local things to a [microsoft azure IOT Hub]
-(https://azure.microsoft.com/en-us/services/iot-hub/).
-This IOT building block resides in the azure cloud and allows 2 way communication with your devices and your IOT infrastructure in the cloud.
-The IOT hub can be connected to other azure services like a [database] (https://azure.microsoft.com/en-us/services/hdinsight/), [stream analytics] (https://azure.microsoft.com/en-us/services/stream-analytics/), [machine learning] (https://azure.microsoft.com/en-us/services/machine-learning/), [time series insights] (https://azure.microsoft.com/en-us/services/time-series-insights/)
+The Azure IoT Hub connector replicates your local things to a [Microsoft Azure IoT Hub](https://azure.microsoft.com/en-us/services/iot-hub/).
+This IoT building block resides in the azure cloud and allows 2 way communication with your devices and your IoT infrastructure in the cloud.
+The IoT hub can be connected to other Azure services like a [database](https://azure.microsoft.com/en-us/services/hdinsight/), [stream analytics](https://azure.microsoft.com/en-us/services/stream-analytics/), [machine learning](https://azure.microsoft.com/en-us/services/machine-learning/) and [time series insights](https://azure.microsoft.com/en-us/services/time-series-insights/).
 
 ## Pricing
 
-You may send up to 8000 messages a day for free. that is 5.5 status updates every minute. Anything more then this in the free price tier is simply neglected by the IOT hub.
-If you want to log more than this, you can switch to a paying price tier: [pricing details] (https://azure.microsoft.com/en-us/pricing/details/iot-hub/)
+You may send up to 8000 messages a day for free. That is 5.5 status updates every minute. Anything more then this in the free price tier is simply neglected by the IoT Hub.
```

Comment:
"more then this" -> "more than this"
Can you also add a new line at the end of each sentence?